functions.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+function checkSingleCardAndGetAllInfo(string $cardTitle): array {
```

Comment:
a better name for the function would be `getCardByTitleFromDb` or something like that

Reply:
This function has been deleted.